### DISCUSSION TITLE
Any ETA for releasing new version?

### DISCUSSION THREAD

**Discussion by oxoocoffee** [hydrabus/hydrafw#135] [General]
I was wondering what are the plans for new gold or beta build past current release?
Perhaps beta release would be good so we could load and test?

cheers

**Comment by bvernoux** [comment]
For information actual [HydraFW v0.10](https://github.com/hydrabus/hydrafw/releases/tag/v0.10) is not a beta and considered as stable since lot of time and it is rock solid without any issue reported.

- We shall check with @Baldanos what will be included in future version and potentially next version could be a v1.0.0 (and not a v0.11) to clarify it is stable and not something in progress
  - See https://github.com/hydrabus/hydrafw/wiki/Task-List#hydrafw-v011-work-in-progress
- Actual issues like https://github.com/hydrabus/hydrafw/issues/49 `PulseView oscilloscope` & https://github.com/hydrabus/hydrafw/issues/21 `USB sniffer capability` are not in development and not really planned to be released one day by core team so contributions are welcome if some people are interested in those features
  - To be discussed if those issues are cancelled (as they are not active) or if someone want to work on them
- Other idea in https://github.com/hydrabus/hydrafw/wiki/Task-List#hydrafw-todoidea shall be discussed too as lot of features are waiting for contributor and feedback

**Comment by bvernoux** [comment]
We have good news HydraFW v0.11 is released for more details see https://github.com/hydrabus/hydrafw/releases/tag/v0.11
